modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsNorthControllerEpicsAo.scala

```diff
@@ -94,14 +95,17 @@ object TcsNorthControllerEpicsAo {
 
     def sysConfig(current: EpicsTcsAoConfig): IO[EpicsTcsAoConfig] = {
       val params = configParams(current)
+      val stabilizationTime = tcs.tc.offsetA
```

Comment:
This seems duplicated across north/south, could it be factored out?

Reply:
Done.